finetune/target_models/masked_language_model.py

```diff
@@ -88,6 +92,9 @@ class MaskedLanguageModel(BaseModel):
     :param config: A :py:class:`finetune.config.Settings` object or None (for default config).
     :param \**kwargs: key-value pairs of config items to override.
     """
+
+    defaults = dict(low_memory_mode=True, batch_size=48, xla=True)
```

Comment:
oh so we did see speed ups with xla = True?

Reply:
Yeah, there is a non-negligible overhead for compilation though, so maybe it doesn't make sense for short runs/debugging.